luminouxx/DocuDocu
# [Dependency+] Add Dependency: Swift Composable Architecture

## Changes
Add ``` Swift Composable Architecture v1.6.1 ```
+ 088e4a3 [Dependency+] Add Dependency: Swift Composable Architecture

## Reason for Changes
+ enhancing project's status management
+ TCA's Test

## Related Issue
+ #6 

## How to Test
1. Clean
``` bash
 tuist clean
```
2. Install & Check Log
``` bash
 tuist install
```
3. Generate & Check
``` bash
 tuist generate
```
